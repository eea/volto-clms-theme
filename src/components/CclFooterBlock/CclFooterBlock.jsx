import React from 'react';
import { Grid } from 'semantic-ui-react';
import classNames from 'classnames';

import './CclFooterBlock.less';

function CclFooterBlock(props) {
  var { children, title, borderBottom = true } = props;
  let titleClass = classNames({
    'footer-clms-col-title': true,
    'border-bottom': borderBottom,
  });
  return (
    <div className={'footer-clms-col'}>
      <div className={titleClass}>
        <h3>{title}</h3>
      </div>
      {children}
    </div>
  );
}

export default CclFooterBlock;
