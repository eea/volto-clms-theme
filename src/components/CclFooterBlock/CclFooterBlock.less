.footer-clms-col {
    background-color: #a0b128;
    color: white;
    width: 28%;
}

.footer-clms-col-title {
  font-size: 1.5rem;
  padding-bottom: 0.5rem;
}

.footer-clms-col-title.border-bottom{
  border-bottom: 1px solid #dadada;
}

@media (max-width: 900px) {
    .footer-clms-col{
      width: 100%;
    }
  }

