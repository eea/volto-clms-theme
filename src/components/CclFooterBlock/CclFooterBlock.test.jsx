import React from 'react';
import renderer from 'react-test-renderer';
import CclFooterBlock from './CclFooterBlock';

test('Renders', () => {
  const component = renderer.create(
    <CclFooterBlock title="Footer block title">
      <p>children component</p>
    </CclFooterBlock>,
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});
