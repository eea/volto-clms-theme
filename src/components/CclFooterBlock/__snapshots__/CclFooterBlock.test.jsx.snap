// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<div
  className="footer-clms-col"
>
  <div
    className="footer-clms-col-title border-bottom"
  >
    <h3>
      Footer block title
    </h3>
  </div>
  <p>
    children component
  </p>
</div>
`;
